app/src/main/res/layout/details_fragment.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingBottom="@dimen/activity_vertical_margin"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    tools:context=".fragments.SubscribersDetailFragment"
+    tools:showIn="@layout/activity_details">
+
+    <ImageView
+        android:layout_width="@dimen/image_view_details_layout"
+        android:layout_height="@dimen/image_view_details_layout"
+        android:id="@+id/image_avatar_detail"
```

Comment:
Please always put the id of the layout on top of the attributes so it is easier to identify. Also for the names, use a abreviations like `img`